packages/legend-graph/src/graph-manager/action/query/Query.ts

```diff
@@ -29,6 +29,11 @@ export class QueryStereotype {
   stereotype!: string;
 }
 
+export class QueryParameterValue {
```

Comment:
I think we should rename this to `QueryDefaultParameterValue` to be clearer
```suggestion
export class QueryDefaultParameterValue {
```